examples/examples/custom_error.rs

```diff
@@ -50,11 +50,11 @@ fn do_another_thing(
 
 #[tracing::instrument]
 fn main() {
-    let subscriber = Registry::default()
-        .with(FmtLayer::default())
+    tracing_subscriber::registry()
```

Comment:
I wish there was a way to replace this expr with something like `tracing_subscriber::fmt()`, aka some sort of conversion fn for a fmt builder to a registry + layer